src/test/java/eu/dissco/core/translator/terms/specimen/CollectorTest.java

```diff
@@ -0,0 +1,99 @@
+package eu.dissco.core.translator.terms.specimen;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.gbif.dwc.ArchiveField;
+import org.gbif.dwc.ArchiveFile;
+import org.gbif.dwc.record.Record;
+import org.gbif.dwc.terms.DwcTerm;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class CollectorTest {
+
+  private static final String COLLECTOR_NAME = "Fricke, Ronald";
+
+  @Mock
+  private ArchiveFile archiveFile;
+  @Mock
+  private Record rec;
+  private Collector collector;
+
+  @BeforeEach
+  void setup() {
+    collector = new Collector();
+  }
+
```

Comment:
Could you put this in the declaration and make it final?

Reply:
Yes thanks, changed it.